src/librustdoc/html/markdown.rs

```diff
@@ -438,6 +440,18 @@ pub fn find_testable_code(doc: &str, tests: &mut ::test::Collector) {
     }
 }
 
+fn get_unstable_features_setting() -> bool {
```

Comment:
I have a feeling it will be much shorter if you call that one instead! It's also best to not reproduce logic wherever possible.